include/clspv/Option.h

```diff
@@ -168,6 +165,18 @@ static bool NonUniformNDRangeSupported() {
          ((Language() == SourceLanguage::OpenCL_C_20));
 }
 
+enum class StorageClass : int {
+  kSSBO = 0,
```

Comment:
These are not quite identical to storage class as a SPIR-V concept.  :-/

But close enough for your use.
Really it's "kinds of storage used for passing directly accessible laid-out data".  But that doesn't have a good name.

no request for change